tools/dev/restart_dev_environment.sh

```diff
@@ -44,11 +47,11 @@ if [[ "$*" == *--s3* ]]; then
   ADDITIONAL_SERVICES+=('storage-aws-s3')
 fi
 
-docker compose -f $DOCKER_COMPOSE_FILE down --remove-orphans
```

Comment:
please revert to `docker compose` instead of `docker-compose`

Reply:
Done